source/backend/routes/users.js

```diff
@@ -86,4 +88,45 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /auth/signup */
+/* Creates new user in database with username, password and email -> unique user/email 
+required */
+router.post('/auth/signup', async (req, res) => {
+  try{
+    const { username } = req.username;
+    const { password } = req.password;
+    const hashedPwd = await bcrypt.hash(password, 10);
+    const { email } = req.email;
+
+    /* check username and email are unique */
+    const checkUser = await usersModel.getByUsername(username);
+    const checkEmail = await usersModel.getByEmail(email);
+
+    if (checkUser.length != 0) {
+      // not sure if status number is correct, 409 = conflict
```

Comment:
409 is usually for PUT requests. Specifically when the change has version conflicts per https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409. Usually, a 401 forbidden will suffice our purpose.